CONTRIBUTING.md

```diff
@@ -22,12 +22,12 @@ value contributions but also value respectful interactions with stravalib users.
 
 **Please be considerate and respectful of others** in all of your communications
 in this repository.
-Everyone must abide by our [Code of Conduct](https://www.github.com/stravalib/stravalib/CODE_OF_CONDUCT.md). Please
+Everyone must abide by our [Code of Conduct](https://github.com/stravalib/stravalib/blob/master/CODE_OF_CONDUCT.md). Please
```

Comment:
i may embed the COC  into the docs for cleaner reading in another PR but not here.